oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingMutationService.java

```diff
@@ -150,9 +151,13 @@ private String checkCreateRequest(EBPFProfilingTaskFixedTimeCreationRequest requ
 
         // query exist processes
         final List<EBPFProfilingTask> tasks = getProcessProfilingTaskDAO().queryTasks(
-                Arrays.asList(request.getServiceId()), request.getTargetType(), calculateStartTime(request), 0);
+                Arrays.asList(request.getServiceId()), request.getTargetType(), request.getStartTime(), 0);
         if (CollectionUtils.isNotEmpty(tasks)) {
-            return "already have profiling task at this time";
+            final EBPFProfilingTask mostRecentTask = tasks.stream()
+                    .min(Comparator.comparingLong(EBPFProfilingTask::getTaskStartTime)).get();
+            if (mostRecentTask.getTaskStartTime() < calculateStartTime(request)) {
+                return "already have profiling most recent task at this time";
```

Comment:
Is this checking existing tasks? Or something? `Task's time range overlaps with other tasks` seems a better response to show. And you should take care of grammar, upper case, and complete sentences.

Reply:
Updated.